traffic_portal/app/src/common/modules/widget/dashboardStats/WidgetDashboardStatsController.js

```diff
@@ -64,9 +64,18 @@ var WidgetDashboardStatsController = function($scope, $interval, $filter, locati
 	};
 
 	var getServerCount = function() {
-		serverService.getEdgeStatusCount()
+		serverService.getServers()
 			.then(function(result) {
-				serverCount = result;
+				serverCount.set("ONLINE", 0);
```

Comment:
this doesn't seem to be working. i know i have 6 reported servers and it shows `0` in the UI.

Reply:
hmm... worked for me. I'll look for it tomorrow

Reply:
Ah, I had a typo left over from when I had to convert it from a `for...of` loop to a `for...in`. Should be working now.